modules/consentManagement.js

```diff
@@ -343,6 +343,9 @@ export function resetConsentData() {
  * @param {object} config required; consentManagement module config settings; cmp (string), timeout (int), allowAuctionWithoutConsent (boolean)
  */
 export function setConsentConfig(config) {
+  // if `config.gdpr` or `config.usp` exist, assume new config format.
+  // else for backward compatability, just use `config`
+  config = config.gdpr || config.usp ? config.gdpr : config;
```

Comment:
Why checking config.usp here? That doesn't seem right.

Reply:
As a separate question from the above - what is the expected use-case around someone who has both GDPR and USP modules together but only specified the USP in their config code?

From looking at the current implementation here, it seems we'll end-up setting the internal defaults for the GDPR module and try to make a call the GDPR CMP.  Is that expected?  Or should it only call the USP CMP because that's what the pub had setup in their config?

Reply:
@bretg - the idea is that if they have either `gdpr` or `usp`, they are using the new format.  they shouldn't mix `usp` with the old `gdpr` config format.

@jsnellbaker - I can imagine a publisher might build a single wrapper (including both gdpr and usp consent modules) for multiple sites, one where they care about gdpr and one where they care about usp.  Or maybe depending on where the user is coming from they may decide to active one or the other.

So I was thinking we should only call the CMPs (gdpr or usp) that are configured.  I think this is backward compatible because if they have the old format (no `gdpr` or `ups` params specified), that means gdpr is configured.